scripts/live_prep.sh

```diff
@@ -0,0 +1,181 @@
+# !/bin/bash
+
+APP_NAME=liveController
+PRODUCT_ROOT_PATH="/opt/kaltura/builds/${APP_NAME}"
+BUILD_CONF=Release
+BUILD_TARGET=node_addons
+SCRIPT_PATH=`dirname ${BASH_SOURCE[0]}`
+
+ADDONS_ROOT_PATH=
+ADDONS_BUILD_PATH=build/Release
+ADDONS_BIN_PATH=../../bin
+
+FORMAT_CONVERTER_BIN=FormatConverter.so
+FFMPEG_BUILD_PATH=~/
+FFMPEG_BUILD_PATH_SET=0
+
+OS=`uname`
+echo "OS=$OS"
+
+contains_element () {
+  local e
+  for e in "${@:2}"; do [[ "$e" == "$1" ]] && return 0; done
+  return 1
+}
+
+while [ "$1" != "" ]; do
```

Comment:
this entire while code is a bit out of scope....   we can just check number of params and return error if something is missing